core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -174,6 +177,14 @@ public Builder trustStoreType(KeyStoreType trustStoreType) {
             return this;
         }
 
+        /**
+         * Errors during hostname verification will be ignored.
+         */
+        public Builder turnOffHostnameVerification() {
```

Comment:
If hostname does not match, there are no errors when native management protocol is used. I will add javadoc, that this is valid only for `ManagementProtocol.HTTPS`